librz/include/rz_core.h

```diff
@@ -1138,6 +1138,14 @@ typedef enum {
 	RZ_CORE_TASK_STATE_DONE
 } RzTaskState;
 
+typedef enum {
+	RZ_CORE_TASK_JOIN_ERR_SUCCESS = 0,
+	RZ_CORE_TASK_JOIN_ERR_CURRENT,
+	RZ_CORE_TASK_JOIN_ERR_NO_TASK,
+	RZ_CORE_TASK_JOIN_ERR_NO_SEM_SINGLE,
+	RZ_CORE_TASK_JOIN_ERR_NO_SEM_GROUP
+} RzCoreTaskJoinErr;
```

Comment:
maybe add a command to debug this?

Reply:
example?

Reply:
i mean, it gets returned but it is not used anywhere. maybe is worth logging it via RZ_LOG_DEBUG or similar.

Reply:
i agree that logging this can be useful ... currently, this is a symptom and not the cause though